src/cplscheme/tests/ExplicitCouplingSchemeTest.cpp

```diff
@@ -172,7 +172,7 @@ void runExplicitCouplingWithSubcycling(
       if (computedTimesteps % 2 == 0) {
         // Data exchange takes only place at every second local timestep,
```

Comment:
Here, the term "timestep" again looks correct. The word "local" can be dropped, since the term "timestep" now always refers to the local timestep and the term "window" refers to what previously was referred to as global (?) or coupling timestep.